MAUI/DataForm/floating-label-layout.md

```diff
@@ -0,0 +1,777 @@
+<dataForm:SfDataForm x:Name="dataForm" 
+                        LayoutType="TextInputLayout"
+                        DataObject="{Binding ContactInfo}"
+                        GenerateDataFormItem="OnGenerateDataFormItem">
+    <dataForm:SfDataForm.TextInputLayoutSettings>
+        <dataForm:TextInputLayoutSettings FocusedStroke="Green"/>
+    </dataForm:SfDataForm.TextInputLayoutSettings>
+</dataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+{
+    if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name")
+    {
+        e.DataFormItem.LayoutType = DataFormLayoutType.TextInputLayout;
+        e.DataFormItem.TextInputLayoutSettings = new TextInputLayoutSettings
+        {
+            FocusedStroke = Colors.Green,
+        };
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Focused and unfocused stroke thickness
+When the editor's view is focused and unfocused, the border's thickness can be changed using the `FocusedStrokeThickness` and `UnfocusedStrokeThickness` properties of `TextInputLayoutSettings`.
+
+{% tabs %}
+{% highlight XAML %}
+<ContentPage 
+...
+xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+<dataForm:SfDataForm x:Name="dataForm" 
+                        LayoutType="TextInputLayout"
+                        DataObject="{Binding ContactInfo}"
+                        GenerateDataFormItem="OnGenerateDataFormItem">
+    <dataForm:SfDataForm.TextInputLayoutSettings>
+        <dataForm:TextInputLayoutSettings FocusedStrokeThickness="3.0"
+                                    UnfocusedStrokeThickness="2.0"/>
+    </dataForm:SfDataForm.TextInputLayoutSettings>
+</dataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+{
+    if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name")
+    {
+        e.DataFormItem.LayoutType = DataFormLayoutType.TextInputLayout;
+        e.DataFormItem.TextInputLayoutSettings = new TextInputLayoutSettings
+        {
+            FocusedStrokeThickness = 3.0,
+            UnfocusedStrokeThickness = 2.0,
+        };
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Unsupported editors
+
+The Floating label layout do not support non-editable editors such as `RadioGroup editor`, `CheckBox editor` and `Switch editor`. Picker editors such as `Picker editor`, `Date picker` and `Time picker` will be updated to support floating label layout.
+
+N> By default the layout type of unsupported editors is `Default`. You can also set layout type for unsupported editor using this `Changing layout type of the DataFormItem`
```

Comment:
N> By default, the layout type of unsupported editors is `Default.` You can also set the layout type for the unsupported editor using this `Changing layout type of the DataFormItem.`

Reply:
Content changed